tests/ios/xcodeproj/Single-Static-Framework-Project.xcodeproj/bazelinstallers/dsyms.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -euo pipefail
+
+echo "Start copying dSYMs at `date`"
+
+# Typicaly dSYMs would only be copied when profiling (and are required by Instruments).
+# In this case, we aren't sure what confiuration is used for profiling,
+# so just copy them over if they exist.
+
+readonly dsym_input="bazel-bin/$BAZEL_BIN_SUBDIR/${FULL_PRODUCT_NAME}.dSYM"
```

Comment:
FWIW XCHammer / Tulsi is reading this out of the BEP: https://github.com/bazelbuild/tulsi/blob/1542ca49297d36c79204f08936eb49ec1af4ac02/src/TulsiGenerator/Scripts/bazel_build.py#L1256

I think with rules_ios this is OK if we can assume this path is guaranteed to match 👍

Reply:
I'm not 100% sure if this code is handling all the cases that logic is: e.g. if you've got child bundles or if the app is  partitioned into into dylibs . I don't know how this code would be updated to handle that case.

Reply:
You're right that this doesn't handle dylibs. I might ping you offline if that's alright.